File: frontend/lib/requests/memory_create_request.dart
class MemoryCreateRequest {
  final String base64image;
  final String name;
  final int creatorId;
  final int journeyId;

  MemoryCreateRequest(
      {required this.base64image,
      required this.name,
      required this.creatorId,
      required this.journeyId});

  Map<String, dynamic> toJson() {
    return {
      'base64image': base64image,
      'name': name,
      'creatorId': creatorId,
      'journeyId': journeyId,
    };
  }
}

File: Backend/Iteramico/DomainService/Requests/Memory/MemoryCreateRequest.cs
using General.Request;

namespace DomainService.Requests.Memory
{
#nullable disable
    public class MemoryCreateRequest : IGeneralRequest
    {
        public string Base64image { get; set; }
        public string Name { get; set; }
        public int CreatorId { get; set; }
        public int JourneyId { get; set; }
    }
}